### ISSUE TITLE
Compliance as a Service

### ISSUE THREAD

**Issue by kevingentile** [cesmii/i3X#10]
In an effort to motivate participation, I propose that authors should consider structuring any requirements in a format that would enable an automated test suite to measure and certify compliance. 

Below are projects across various technology domains which provide compliance oriented testbeds:

* [cncf/k8s-conformance](https://github.com/cncf/k8s-conformance)
* [google/x509test](https://github.com/google/x509test)
* [usnistgov/macos_security](https://github.com/usnistgov/macos_security)
* [prometheus/compliance](https://github.com/prometheus/compliance)
* [ThalesGroup/riscv-compliance](https://github.com/ThalesGroup/riscv-compliance)
* [terraform-compliance/cli](https://github.com/terraform-compliance/cli)
* [OpenSCAP/openscap](https://github.com/OpenSCAP/openscap)
* [google/openpgp-interop](https://github.com/google/openpgp-interop)

**Comment by jwise-mfg** [comment]
Yup, this is a good idea. The Working Group kicked-off today and discussed scope that will include an API test endpoint and client. A test suite makes sense too, marking as a feature request for consideration.

**Comment by jwise-mfg** [comment]
Fixed with https://www.i3x.dev/conformance

PRs welcome to continue to improve that test suite.
